arm-advisor/2017-03-31/swagger/advisor.json

```diff
@@ -0,0 +1,679 @@
+						"description": "The name of the suppression.",
+						"required": true,
+						"type": "string"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "The recommendation has been activated."
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"DeleteSuppression": {
+						"$ref": "../examples/DeleteSuppression.json"
+					}
+				}
+			}
+		},
+		"/{resourceUri}/providers/Microsoft.Advisor/recommendations/{recommendationId}": {
+			"get": {
+				"tags": [
+					"GetRecommendations"
+				],
+				"description": "Obtains details of a cached recommendation.",
+				"operationId": "Recommendations_Get",
+				"parameters": [
+					{
+						"name": "resourceUri",
+						"in": "path",
+						"description": "The fully qualified Azure Resource Manager identifier of the resource to which the recommendation applies.",
+						"required": true,
+						"type": "string"
+					},
+					{
+						"name": "recommendationId",
+						"in": "path",
+						"description": "The recommendation ID.",
+						"required": true,
+						"type": "string"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationBase"
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"GetRecommendationDetail": {
+						"$ref": "../examples/GetRecommendationDetail.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/suppressions": {
+			"get": {
+				"tags": [
+					"Suppressions"
+				],
+				"description": "Retrieves the list of snoozed or dismissed suppressions for a subscription. The snoozed or dismissed attribute of a recommendation is referred to as a suppression.",
+				"operationId": "Suppressions_List",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK.",
+						"schema": {
+							"type": "array",
+							"items": {
+								"$ref": "#/definitions/SuppressionContract"
+							}
+						}
+					}
+				},
+				"deprecated": false,
+				"x-ms-examples": {
+					"ListSuppressions": {
+						"$ref": "../examples/ListSuppressions.json"
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"ResourceRecommendationBaseListResult": {
+			"description": "The list of Advisor recommendations.",
+			"type": "object",
+			"properties": {
+				"nextLink": {
+					"description": "The link used to get the next page of recommendations.",
+					"type": "string"
+				},
+				"value": {
+					"description": "The list of recommendations.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/ResourceRecommendationBase"
+					}
+				}
+			}
+		},
+		"ResourceRecommendationBase": {
```

Comment:
Can you confirm if this is a resource? If so, it should have an allOf of a resource.